### PR TITLE
Enable flake8

### PR DESCRIPTION
**Pull request by yazug**
Fixes Issue https://github.com/release-depot/koji_wrapper/issues/21

### CODE REVIEW

**Comment by yazug** [comment]
@jguiditta @fuzzball81 @lhh Feedback welcome on minimal change enabling flake8 for all the sources and correcting the autopep8 whitespace cleanup and other linting issues found.

File: tox.ini [hunk 1]
```diff
@@ -16,6 +16,6 @@ commands =
     pipenv install --dev
     pipenv run flake8 --version
     pipenv run py.test tests
-    pipenv run flake8 setup.py docs project test
-
+    pipenv run flake8 --ignore E501 setup.py koji_wrapper/ tests/
```
[the remark this answers is not in the document]
Comment (lhh) [reply tox.ini]: Took a minute to see what you meant. That makes sense. Hard to contextually insert stuff in this UI.

[the remark this answers is not in the document]
Comment (yazug) [reply tox.ini]: meh, at this point it is passing linting, that's a design decision on how to rotate things to improve them going forward.  This pull request is trying to get linting enabled again for the source code and clean, without re-factoring things.

File: tox.ini [hunk 2]
```diff
@@ -16,6 +16,6 @@ commands =
...
+    pipenv run flake8 --ignore E501,E402 setup.py docs/
```
Comment (yazug) [inline tox.ini]: Hmm setup.py is in 2 entries.

Comment (yazug) [reply tox.ini]: fixed

File: tox.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (fuzzball81) [reply tox.ini]: You don't need the trailing '/' but its not going to hurt anything.

**Comment by jguiditta** [comment]
Jon, can you rebase this to master when you have a chance?  I merged your other patch, so this one has a conflict now.

**Pull request merged**